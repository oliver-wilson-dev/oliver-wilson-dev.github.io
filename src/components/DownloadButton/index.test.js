import React from 'react';
import { shallow } from 'enzyme';
import DeleteButton from '.';

jest.mock('./download.svg', () => {
  const DownloadButtonIcon = () => null;

  return DownloadButtonIcon;
});

const render = () => shallow(<DeleteButton />);

describe('<DeleteButton/>', () => {
  it('should exist', () => {
    expect(render().exists()).toBe(true);
  });

  it('should render correctly', () => {
    expect(render()).toMatchSnapshot();
  });
});
