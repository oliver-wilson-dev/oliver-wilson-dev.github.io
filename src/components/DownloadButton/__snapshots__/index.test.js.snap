// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DeleteButton/> should render correctly 1`] = `
<div
  className="downloadButtonContainer flexCenter"
>
  <a
    className="downloadButton"
    download="Oliver Wilson Curriculum Vitae.pdf"
    href="public/documents/CV.pdf"
    rel="noopener noreferrer"
    target="_blank"
  >
    download my cv
  </a>
</div>
`;
